src/views/templates/list/VirtualMachineTemplatesList.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { useVmTemplates } from '@kubevirt-utils/resources/template';
+import {
+  K8sResourceCommon,
+  ListPageBody,
+  ListPageFilter,
+  ListPageHeader,
+  useListPageFilter,
+  VirtualizedTable,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+import { useFilters } from '../utils/virtualMachineTemplatesRowFilter';
+
+import useVirtualMachineTemplatesColumns from './components/VirtualMachineTemplatesColumns';
+import VirtualMachineTemplatesRow from './components/VirtualMachineTemplatesRow';
+import VirtualMachineTemplateSupport from './components/VirtualMachineTemplateSupport';
+
+const VirtualMachineTemplatesList: React.FC = () => {
+  const { t } = useKubevirtTranslation();
+  const { templates, loaded, loadError } = useVmTemplates();
+  const [data, filteredData, onFilterChange] = useListPageFilter(templates, useFilters());
+
+  return (
+    <>
+      <ListPageHeader title={t('Virtual Machine Templates')}></ListPageHeader>
+      <ListPageBody>
+        <ListPageFilter
+          data={data}
+          loaded={loaded}
+          rowFilters={useFilters()}
+          onFilterChange={onFilterChange}
+        />
+        <VirtualMachineTemplateSupport />
+        <VirtualizedTable<K8sResourceCommon>
+          data={filteredData}
+          unfilteredData={data}
+          loaded={loaded}
+          loadError={loadError}
+          columns={useVirtualMachineTemplatesColumns()}
+          Row={VirtualMachineTemplatesRow}
+          rowData={{ kind: 'Template' }}
```

Comment:
I think it would be better to use `TemplateModel.kind` instead of hard coded const